File: lib/classes.rb
class Book 
    attr_accessor :title, :author, :pages
    def initialize(title, author, pages)
        @title = title
        @author = author
        @pages = pages
    end 
end 

book1 = Book.new("Harry Porter", "JK Rowling", 400)

puts book1.title
puts book1.author
puts book1.pages

File: lib/classes.py
class Book:
    def __init__(self, title, author, pages):
        self.title = title
        self.author = author
        self.pages = pages


book1 = Book("Harry Porter", "JK Rowlings", 400)

print(book1.title)
print(book1.author)
print(book1.pages)
